### PR TITLE
Login

### PR DESCRIPTION
**Pull request by babbtx**

### CODE REVIEW

File: dist/spiceworks-sdk-0.0.2.js [hunk 1]
```diff
@@ -3222,7 +3222,7 @@ define("oasis/sandbox",
 
       pkg = pkg || {};
 
-      this.adapter = options.adapter || new iframeAdapter();
+      this.adapter = options.adapter || Oasis.adapters.iframe;
```
Comment (babbtx) [inline dist/spiceworks-sdk-0.0.2.js]: I didn't do this, to my knowledge. Is this a local artifact?

File: dist/spiceworks-sdk-host.js [hunk 2]
```diff
@@ -3222,7 +3222,7 @@ define("oasis/sandbox",
 
       pkg = pkg || {};
 
-      this.adapter = options.adapter || new iframeAdapter();
+      this.adapter = options.adapter || Oasis.adapters.iframe;
```
Comment (babbtx) [inline dist/spiceworks-sdk-host.js]: same here.

File: dist/spiceworks-sdk-0.0.2.js (hunk 1 above)
Comment (msgerbush) [inline dist/spiceworks-sdk-0.0.2.js]: yeah, I think we actually need the top version.  this is the shitty part about having our compiled code as part of the git repo.  this comes from building the library with the wrong version of Oasis.  so if you build without running `bower install`, the concatenated files could include an old version of our dependencies :disappointed:.

Comment (msgerbush) [inline dist/spiceworks-sdk-0.0.2.js]: i'm taking care of it now.  i think the version specified in bower.json was actually old.

**Comment by msgerbush** [comment]
I pulled in the updated Oasis version and rebuilt the libraries, so those diffs in Oasis went away.  Everything else looks good :+1: .  Merging.

**Pull request merged**